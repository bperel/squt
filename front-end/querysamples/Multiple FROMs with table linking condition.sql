SELECT co.contractID AS ID, co.contractOrigin, comp.companyName
FROM Contract co, Company comp
WHERE co.contractType='C' AND co.contractID > 0
AND co.companyCode = comp.companyCode
ORDER BY co.contractDate;