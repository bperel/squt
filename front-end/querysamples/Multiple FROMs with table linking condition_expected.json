{
   "Functions" : {
      "1" : {
         "to" : "NOWHERE",
         "name" : ">",
         "Constants" : {
            "0" : "0"
         }
      },
      "0" : {
         "to" : "NOWHERE",
         "name" : "=",
         "Constants" : {
            "C" : "C"
         }
      },
      "2" : {
         "to" : "NOWHERE",
         "name" : "="
      }
   },
   "Tables" : {
      "Company" : {
         "comp" : {
            "OUTPUT" : {
               "companyCode" : {
                  "2" : "companyCode"
               },
               "companyName" : {
                  "-1" : "companyName"
               }
            }
         }
      },
      "Contract" : {
         "co" : {
            "OUTPUT" : {
               "companyCode" : {
                  "2" : "companyCode"
               },
               "contractOrigin" : {
                  "-1" : "contractOrigin"
               },
               "contractType" : {
                  "0" : "contractType"
               },
               "contractID" : {
                  "1" : "contractID",
                  "-1" : "ID"
               }
            },
            "SORT" : {
               "contractDate" : "ASC"
            }
         }
      }
   }
}
